components/molecules/CollaborationCard/collaboration-card.tsx

```diff
@@ -5,7 +5,7 @@ import { getAvatarByUsername } from "lib/utils/github";
 import Button from "components/atoms/Button/button";
 
 export interface CollaborationRequestObject extends React.ComponentProps<"div"> {
-  requestor: DbUser | undefined;
```

Comment:
This is fine as `DbUser` only. It had a union with `undefined` so that it would work with Storybook. Now Storybook uses a mocked requestor user.